nifi-frontend/src/main/frontend/apps/update-attribute/src/app/pages/update-attribute/ui/rule-listing/rule-listing.component.html

```diff
@@ -0,0 +1,175 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+@if (rulesList.length === 0 && !newRule) {
+    <div class="h-full flex justify-center items-center gap-x-4">
+        @if (isEditable) {
+            <div>No rules configured for this Processor. Get started by defining your first.</div>
+            <button mat-flat-button (click)="populateNewRule()">Create Rule</button>
+        } @else {
+            <div>No rules configured for this Processor.</div>
+        }
+    </div>
+} @else {
+    <div class="rule-listing flex flex-col gap-y-4">
+        @if (rulesList.length > 0 || !newRule) {
+            <div class="flex flex-col gap-y-1">
+                <div class="flex gap-x-1 items-center">
+                    <div>Use original FlowFile for matching rules</div>
+                    <form [formGroup]="flowFilePolicyForm">
+                        <mat-slide-toggle
+                            formControlName="useOriginalFlowFilePolicy"
+                            (change)="flowFilePolicyToggled($event)"></mat-slide-toggle>
+                    </form>
+                </div>
+                <div class="text-sm">If disabled rules will be executed using a clone of the original FlowFile</div>
```

Comment:
This is incorrect. If disabled, ordering doesn't matter because each matching rule will use a clone of the original FlowFile. Ordering only matters when executing using the original FlowFile.